Description, Name, Mapping and Access related changes all throughout the codebase.

HaultyAnonie opened the pull request:
## About the Pull Request

Several quality of life changes, removal of quirky descriptions mentioning the galaxy and overall space-craft, most prevalent in hydroponics and drinks alike.
Tried to edit the supply system cost, but it bugged out- so that's a no no.
Fixed up signs a bit, added a medical diploma to the chief medical officer. 
Our emergency box in our bag has always been named 'crew survival kit.' this is not a crew, we are not on a station.

## Why It's Good For The Game

Easier to understand for our non-English speakers.
Better descriptions, more immersed- gives the items a more refined aspect- fixes contradictions.
Things do as they say and work as they are meant to.

## Changelog

:cl:
code: refined the description and name of objects as well as the most common one; which is the 'crew survival box' in everyone's bag.
tweak: gives the Guard Commander access to engineering, but only level one- same as EZ agents.
tweak: removes part of the hydroponic tray's advanced description involving spaceships and the galaxy.
fix: fixes https://github.com/Foundation-19/Foundation-19/issues/1247
fix: fixes https://github.com/Foundation-19/Foundation-19/issues/223
tweak: tweaked containment engineers access to allow him to access 106 and 173 but not LCZ and HCZ to get into such places without permission in the first place.
add: added a new pill bottle containing pills that necessarily save but do not heal damage, so cannot be abused for combat and min-maxxed.
add: a medical diploma for the chief medical officer.
add: a pill bottle called (prescribed stabiliser) which has a concoction of pills only available in the LCZ zone for D-Class when medical staff are not around.
add: first-aid closets all around the facility in areas I've usually seen combat near.
tweak: tweaks D-Class mining, allowing them to mine out tunnels from more places than before; less meta locations and makes LCZ guards actually **have** to patrol.
add: The traitorous holdout pistol is now able to be holstered inside boots.
/:cl:

![image](https://github.com/Foundation-19/Foundation-19/assets/131271192/012607cd-2928-442c-9b11-776341ef8da7)
![image](https://github.com/Foundation-19/Foundation-19/assets/131271192/edbcd45e-c7be-4024-84ce-49da5df3938b)


<!--
Examples were changelog entries are optional/not typically required but encouraged:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

As a courtesy, for ported PRs, please include if you have the blessing of the author. While not required, we encourage basic decency in porting others' work. It is also sufficient to note that the author has not expressed displeasure at the idea of their work getting ported.
Please refrain from porting works of authors that have expressed displeasure in having their work ported, thank you.
-->

Commit:
Removes everything that doesn't have immersion.

aidan-bowman commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -96,8 +96,8 @@
 /obj/item/storage/box/survival
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "station survival kit"
```
Station?

HaultyAnonie replied:
mind blurt.

aidan-bowman commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -96,8 +96,8 @@
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "station survival kit"
+	desc = "A box that has an assortment of items necessary for survival. Striped with red and yellow lines to indicate this one has oxygen."
```
This is... poorly worded.

HaultyAnonie replied:
no it's not.

aidan-bowman commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -10,7 +10,7 @@
 /obj/item/clothing/suit/bluetag
 	name = "blue laser tag armour"
-	desc = "Blue Pride, Galaxy Wide."
+	desc = "Blue Pride, World Wide."
```
Poorly worded, also why the fuck do we need a generic blue tag?

HaultyAnonie replied:
because... 'm fixing it.

aidan-bowman commented on code/modules/projectiles/guns/projectile/pistol.dm:
```diff
@@ -89,7 +89,7 @@
 /obj/item/gun/projectile/pistol/throwback
 	name = "pistol"
-	desc = "A product of one of thousands of illegal workshops from around the galaxy. Often replicas of ancient Earth handguns, these guns are usually found in hands of frontier colonists and pirates."
+	desc = "A product of one of thousands of illegal workshops from around the world. Often replicas of ancient handguns, these guns are usually found in hands of pirates and low-lifes."
```
Uhhhhhhh, handguns aren't "ancient" and as far as I know we don't have pirates in the modern day.

HaultyAnonie replied:
yes we do have pirates in the modern day and this needs better wording, it's a draft.

TheDarkElites commented:
bad title, not descriptive of PR

Commit:
Merge branch 'dev' into newsite

Commit:
Please bully the hell out of my code.

I'm trying to learn, somethings wrong so I'm gonna commit.

Commit:
Merge branch 'newsite' of https://github.com/HaultyAnonie/Foundation-19 into newsite

Commit:
I suppose.

Commit:
Fix indentation

thx seven

EgorDinamit commented on code/game/machinery/vending.dm:
```diff
@@ -938,15 +938,16 @@
 		/obj/item/stack/medical/advanced/ointment = 10,
 		/obj/item/stack/medical/splint = 5,
-		/obj/item/reagent_containers/hypospray/autoinjector/pain = 15
+		/obj/item/reagent_containers/hypospray/autoinjector/pain = 15, //tramadol autoinjector
```
```suggestion
		/obj/item/reagent_containers/hypospray/autoinjector/pain = 15, //tramadol autoinjector,
```
If I am not misremembering - Bay's indentation check is just that dumb, yes.

Commit:
Update code/game/machinery/vending.dm

bay... gr

Co-authored-by: EgorDinamit <53223414+EgorDinamit@users.noreply.github.com>

Commit:
attempts to fix my creation.

Commit:
fix fix fix fix

easy

Commit:
Merge branch 'newsite' of https://github.com/HaultyAnonie/Foundation-19 into newsite

Commit:
Containment Engineer: BUG FIX

Commit:
One more fix.

Commit:
fixes the error runtime.

Commit:
Deep within the confines.

Adds a little surprise in DCZ mining as well as multiple medical kits around the site.

Commit:
captain to site...

Commit:
Merge branch 'dev' into newsite

Commit:
more mapping

Commit:
Merge branch 'newsite' of https://github.com/HaultyAnonie/Foundation-19 into newsite

Commit:
Update site53-1.dmm

fixes error.

Commit:
Update site53-1.dmm

im so dumb

Commit:
Update site53-1.dmm

dont know whats going o nbut here

Commit:
Update site53-1.dmm

wtf is going on

Commit:
Update site53-1.dmm

this is why we cant have nice things

Commit:
we cant have nice things

okay pleas

Commit:
Update site53-1.dmm

LEAVE

Commit:
please just go..

Commit:
Update site53-1.dmm

wow, that's where it was, cool.

aidan-bowman commented:
With the amount of unexplained balance changes you've made, there is NO way this is getting merged without atomization

aidan-bowman commented on code/game/objects/structures/signs.dm:
```diff
@@ -466,9 +467,14 @@
 	. = ..()
 	if(claimant)
-		to_chat(user,"This one belongs to Dr.[claimant], MD.")
+		to_chat(user,"This one belongs to Dr. [claimant], Chief Medical Officer.")
```
You don't understand what MD means don't you

HaultyAnonie replied:
medical doctor

aidan-bowman commented on code/game/objects/structures/signs.dm:
```diff
@@ -466,9 +467,14 @@
 
+/obj/item/sign/medipolma/attackby(obj/item/W as obj, mob/user as mob)
+	if(istype(W, /obj/item/tape_roll))
+		user.visible_message(SPAN_WARNING("[user] attaches [W] to the [src]."), SPAN_WARNING("You attach some [W] to the [src]."))
```
??? this doesn't even do anything?

HaultyAnonie replied:
its a draft for a reason.

aidan-bowman commented on code/game/objects/structures/signs.dm:
```diff
@@ -454,7 +455,7 @@
 /obj/item/sign/medipolma
 	name = "medical diploma"
-	desc = "A fancy print laminated paper that certifies that its bearer is indeed a Doctor of Medicine, graduated from a medical school in one of fringe systems. You don't recognize the name though, and half of latin words they used do not actually exist."
+	desc = "A fancy print laminated paper that verifies that the bearer is actually a Doctor of Medicine, graduating from an exemplary medical school, but it additionally features some interesting latin inscriptions."
```
Rewrite this.

HaultyAnonie replied:
why

aidan-bowman commented on code/modules/reagents/reagent_containers/food/drinks/bottle.dm:
```diff
@@ -339,7 +339,7 @@
 /obj/item/reagent_containers/food/drinks/bottle/herbal
 	name = "Liqueur d'Herbe"
-	desc = "A bottle of the seventh-finest herbal liquor sold under a generic name. The back label has a load of guff about the monks who traditionally made this particular variety."
+	desc = "A bottle of the seventh-finest herbal liquor sold popularly. The back label has a load of guff about the monks who traditionally made this particular variety."
```
Remove the "popularly".

HaultyAnonie replied:
ok.

aidan-bowman commented on code/__defines/access_defs.dm:
```diff
@@ -87,6 +87,7 @@
 #define ACCESS_CONTAINMENT_SCP_343 "ACCESS_CONTAINMENT_SCP_343"
 #define ACCESS_CONTAINMENT_SCP_399 "ACCESS_CONTAINMENT_SCP_399"
 #define ACCESS_CONTAINMENT_SCP_409 "ACCESS_CONTAINMENT_SCP_409"
+#define ACCESS_CONTAINMENT_SCP_500 "ACCESS_CONTAINNMENT_SCP_500"
```
Do you even use this define?

HaultyAnonie replied:
no.

aidan-bowman commented on code/game/jobs/job/engineering.dm:
```diff
@@ -168,6 +170,7 @@
 		ACCESS_ENGINEERING_LVL3,
 		ACCESS_ENGINEERING_LVL4,
 		ACCESS_ATMOSPHERICS,
+		ACCESS_NETWORK,
```
Explain.

HaultyAnonie replied:
adds apc access, as shown in the bug report that i fixed

projectkepler-ru replied:
Why do they need specific access?

HaultyAnonie replied:
what do you mean specific access, they need apc access and i gave it to them.

aidan-bowman commented on code/game/jobs/job/security.dm:
```diff
@@ -42,6 +43,7 @@
 		ACCESS_SCIENCE_LVL2,
 		ACCESS_SCIENCE_LVL3,
 		ACCESS_SCIENCE_LVL4,
```
Lower the science access here.

HaultyAnonie replied:
Science access is used for SCP-containment-cell access; the containment engineer wouldn't be able to do his job without this.

aidan-bowman commented on code/game/machinery/_machines_base/machinery.dm:
```diff
@@ -430,7 +430,7 @@ Class Procs:
 		display_parts(user)
 	if(stat & NOSCREEN)
-		to_chat(user, "It is missing a screen, making it hard to interact with.")
+		to_chat(user, "It is missing a screen, making it hard to interact with; but it might not need it.")
```
Unnecessary.

HaultyAnonie replied:
ok but, added.

projectkepler-ru replied:
unnecessary

aidan-bowman commented on code/game/machinery/vending.dm:
```diff
@@ -962,21 +963,22 @@
 /obj/machinery/vending/wallmed1
 	name = "\improper NanoMed"
-	desc = "A wall-mounted version of the NanoMed."
+	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilisation and much more."
```
If you're going to make a vending machine's description unnecessarily long, at LEAST do it to all variants of that vending machine.

HaultyAnonie replied:
this is the one that is used ingame.

aidan-bowman commented on code/game/machinery/vending.dm:
```diff
@@ -962,21 +963,22 @@
+		/obj/item/storage/med_pouch/trauma = 2,
+		/obj/item/storage/med_pouch/burn = 2,
+		/obj/item/storage/med_pouch/oxyloss = 2,
+		/obj/item/storage/med_pouch/toxin = 2,
```
Explain all the balance changes here.

HaultyAnonie replied:
one is not enough- incase reinjection might be needed, two is good enough for people not to abuse this vending machine.

HaultyAnonie replied:
im a medical main, someone snags all these med_pouches first thing.

projectkepler-ru replied:
Bad

HaultyAnonie replied:
how is this bad.

aidan-bowman commented on code/game/machinery/vending.dm:
```diff
@@ -1003,21 +1005,23 @@
 	name = "\improper SecTech"
-	desc = "A security equipment vendor."
-	product_ads = "Crack capitalist skulls!;Beat some heads in!;Don't forget - harm is good!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
+	desc = "A security equipment vendor, with some customary donuts; help yourself."
```
Remove everything after the semicolon.

HaultyAnonie replied:
done.

aidan-bowman commented on code/game/machinery/vending.dm:
```diff
@@ -1003,21 +1005,23 @@
-	desc = "A security equipment vendor."
-	product_ads = "Crack capitalist skulls!;Beat some heads in!;Don't forget - harm is good!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
+	desc = "A security equipment vendor, with some customary donuts; help yourself."
+	product_ads = "Defend yourself!;Help others!;Don't forget - security is a need not a want!;Your weapons are right here.;Handcuffs!;Freeze, scumbag!;Don't tase me bro!;Tase them, bro.;Why not have a donut?"
```
"Don't forget - security is a need not a want!" is fucking stupid. Change the other meme-y quotes to be more befitting as well.

aidan-bowman commented on code/game/machinery/vending.dm:
```diff
@@ -1003,21 +1005,23 @@
 	vend_delay = 14
 	base_type = /obj/machinery/vending/security
-	req_access = list(ACCESS_SECURITY)
+	req_access = list(ACCESS_SECURITY,ACCESS_SECURITY_LVL1)
```
Get rid of the generic ACCESS_SECURITY.

aidan-bowman commented on code/datums/uplink/medical.dm:
Explain the balance changes in this file.

aidan-bowman commented on code/game/objects/items/bodybag.dm:
```diff
@@ -14,8 +14,8 @@
 /obj/item/storage/box/bodybags
-	name = "body bags"
-	desc = "This box contains body bags."
+	name = "\improper box of body bags"
```
\improper isn't needed here

HaultyAnonie replied:
testing showed it somehow did.

aidan-bowman commented on code/game/objects/items/bodybag.dm:
```diff
@@ -25,7 +25,7 @@
 	new /obj/item/bodybag(src)
 	new /obj/item/bodybag(src)
-	new /obj/item/bodybag(src)
+	new /obj/item/wirecutters(src)
```
WHAT THE FUCK DUDE

HaultyAnonie replied:
what, it's a **d r a f t**

aidan-bowman commented on code/game/objects/items/bodybag.dm:
```diff
@@ -14,8 +14,8 @@
-	name = "body bags"
-	desc = "This box contains body bags."
+	name = "\improper box of body bags"
+	desc = "This box consists of body bags, each with a written label, this particular box includes wirecutters to remove labels."
```
WAY too long

HaultyAnonie replied:
body bags arent used as much as you'd expect- always examined.

aidan-bowman commented on code/game/objects/items/devices/radio/headset.dm:
```diff
@@ -193,8 +193,8 @@
 	ks1type = /obj/item/device/encryptionkey/merchant
 
 /obj/item/device/radio/headset/heads/captain
```
Change the path too, while you're at it.

HaultyAnonie replied:
ok.

aidan-bowman commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -96,8 +96,8 @@
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "site survival kit"
+	desc = "A survival box containing a variety of supplies necessary to ensure survival. This one possesses oxygen, as shown by the red panel and two striped yellow lines."
```
Get rid of the "survival" as well as the anecdote about the red panel and yellow stripes. Use "contains" instead of "possesses". Also drop the flowery writing.

HaultyAnonie replied:
it's the most common object in the game, it deserves a detailed description as to why it was in your bag every shift.

HaultyAnonie replied:
changed.

aidan-bowman commented on code/game/objects/items/weapons/storage/firstaid.dm:
Drop the /improper macros.

HaultyAnonie replied:
ok, i was doing some testing.

aidan-bowman commented on code/modules/reagents/reagent_containers/food/drinks/bottle.dm:
```diff
@@ -220,7 +220,7 @@
 /obj/item/reagent_containers/food/drinks/bottle/bottleofnothing
 	name = "Bottle of Nothing"
-	desc = "A bottle filled with nothing."
+	desc = "A bottle filled with nothing, but probably something."
```
Just delete this whole drink lol, we don't have mimes.

HaultyAnonie replied:
elites told me not to delete anything on the codebase, im just making it more friendly for scp13

HaultyAnonie replied:
sorry, deleting this breaks things- someone else can pr the deletion.

projectkepler-ru replied:
You can delete it

aidan-bowman commented on maps/site53/structures/closets/medical.dm:
```diff
@@ -1,5 +1,5 @@
 /*
- * Torch Medical
+ * Torch -/- Medical
```
Delete this entire comment.

HaultyAnonie replied:
okay- the codebase is full of old things, i just wanted to preserve it's oldness like elite said.

aidan-bowman commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -10,7 +10,7 @@
 /obj/item/clothing/suit/bluetag
 	name = "blue laser tag armour"
-	desc = "Blue Pride, Galaxy Wide."
+	desc = "Blue Pride, World Wide."
```
Issue still stands, just make it "Worldwide" its that simple

HaultyAnonie replied:
i didnt even touch the redtag

projectkepler-ru replied:
Yes because that's a suggested change

Commit:
you already know - cheesepizza suggestions

we love it!

Commit:
Update command.dm

i thought i already did this but whatever

Commit:
more!!!

removes the generic access + breathable not insulated because you will still get shocked if you arent wearing gloves

Commit:
removes improper macros

Commit:
Update vending.dm

we love the non memey

Commit:
Update site53-1.dmm

fixes the stabilisers

projectkepler-ru commented on code/datums/uplink/medical.dm:
```diff
@@ -6,36 +6,36 @@
 /datum/uplink_item/item/medical/sinpockets
 	name = "Box of Sin-Pockets"
-	desc = "A box of filled dough pockets. Great for a quick meal when you're hiding from Security. Instructions included on the box."
+	desc = "A box of filled dough pockets. Great for a quick meal. Instructions included on the box."
```
Really should change this as the donk pocket inside it is meant to be a unique traitor item for healing

HaultyAnonie replied:
The sinpockets inside don't contain any special chemical, it's just food.

projectkepler-ru commented on code/datums/uplink/medical.dm:
```diff
@@ -6,36 +6,36 @@
 	name = "Stabilisation First Aid Kit"
 	desc = "Contains variety of emergency medical pouches."
-	item_cost = 16
+	item_cost = 4
```
Why?

HaultyAnonie replied:
because this stabilisation kit is found everywhere, it's the one with all the 5 med_pouches, not even that good and only stabilises.

projectkepler-ru commented on code/datums/uplink/medical.dm:
```diff
@@ -6,36 +6,36 @@
 	name = "Surgery Kit"
 	desc = "Contains all the tools needed for on the spot surgery, assuming you actually know what you're doing with them. Floor sterilization not included."
-	item_cost = 40
+	item_cost = 8
```
that is stupid low

HaultyAnonie replied:
40 out of 120 telecrystals is stupid high as well

projectkepler-ru commented on code/datums/uplink/medical.dm:
```diff
@@ -6,36 +6,36 @@
 	name = "Combat Medical Kit"
 	desc = "Contains most medicines you need to recover from injuries and illnesses, all in a convenient pill form. Splints for broken bones also included!"
-	item_cost = 48
+	item_cost = 6
```
why is a literal medkit full of way more useful stuff cheaper than a surgery kit,
again, the price s way too low

HaultyAnonie replied:
i agree, will balance.

projectkepler-ru commented on code/game/jobs/job/security.dm:
```diff
@@ -28,6 +28,7 @@
 	access = list(
 		ACCESS_COM_COMMS,
 		ACCESS_SEC_COMMS,
+		ACCESS_SECURITY, //this might break something, but if you see this, it probably didnt!
```
did you need to add the comment for that

HaultyAnonie replied:
yes

projectkepler-ru commented on code/game/machinery/vending.dm:
```diff
@@ -938,15 +938,16 @@
 		/obj/item/stack/medical/advanced/ointment = 10,
 		/obj/item/stack/medical/splint = 5,
-		/obj/item/reagent_containers/hypospray/autoinjector/pain = 15
+		/obj/item/reagent_containers/hypospray/autoinjector/pain = 15, //tramadol autoinjector,
```
why the comment?

HaultyAnonie replied:
so people know what autoinjector/pain is for future reference.

projectkepler-ru commented on code/game/machinery/vending.dm:
```diff
@@ -962,21 +963,22 @@
 /obj/machinery/vending/wallmed1
 	name = "\improper NanoMed"
-	desc = "A wall-mounted version of the NanoMed."
+	desc = "A wall-mounted version of the NanoMed, stocked with the equipment necessary for stabilisation and much more."
```
Why?

HaultyAnonie replied:
so it's more explanatory what the contents inside are.

projectkepler-ru commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -96,8 +96,8 @@
 /obj/item/storage/box/survival
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "site survival kit"
```
```suggestion
	name = "personal emergency kit"
```

projectkepler-ru commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -96,8 +96,8 @@
-	name = "crew survival kit"
-	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	name = "site survival kit"
+	desc = "A survival box containing a variety of supplies. This one contains oxygen, as shown by the red panel and two striped yellow lines."
```
```suggestion
	desc = "A box containing a variety of supplies. Contains basic emergency kit, as indicated by the red panel and two striped yellow lines."
```

HaultyAnonie replied:
you didn't even make this grammatically correct.

projectkepler-ru commented on code/game/objects/items/weapons/storage/firstaid.dm:
```diff
@@ -427,6 +427,21 @@
+	startswith = list(
+			/obj/item/reagent_containers/pill/inaprovaline = 6,
+			/obj/item/reagent_containers/pill/oxycodone = 6,
+			/obj/item/reagent_containers/pill/dexalin_plus = 2,
```
now this is just medical powergamer's dream

HaultyAnonie replied:
**it's a draft!!!**

projectkepler-ru commented on code/modules/projectiles/guns/projectile/pistol.dm:
```diff
@@ -89,7 +89,7 @@
 /obj/item/gun/projectile/pistol/throwback
 	name = "pistol"
-	desc = "A product of one of thousands of illegal workshops from around the galaxy. Often replicas of ancient Earth handguns, these guns are usually found in hands of frontier colonists and pirates."
+	desc = "A product of one of the world's hundreds of illicit workshops. These firearms, which are frequently reproductions of past handgun blueprints, are typically seen in the hands of smugglers and low-lifes."
```
```suggestion
	desc = "A product of one of the worlds' hundreds of illicit workshops. These firearms, which are frequently reproductions of past handgun blueprints, are typically seen in the hands of smugglers and low-lifes."
```

aidan-bowman replied:
No don't do that, `worlds'` is the plural possessive which implies there are several worlds.

projectkepler-ru commented on maps/site53/items/clothing/solgov-under.dm:
```diff
@@ -226,7 +226,7 @@
 
 /obj/item/clothing/under/solgov/utility/fleet
 	name = "utility coveralls"
-	desc = "The utility uniform of the SCP Foundation, made from an insulated material."
```
we are in the northern hemisphere and outside of the facility visibly contains snow
not sure why you'd change that line

HaultyAnonie replied:
[Alaska is in both hemispheres](https://en.wikipedia.org/wiki/Geography_of_Alaska)

projectkepler-ru commented on code/game/objects/structures/signs.dm:
```diff
@@ -466,9 +467,14 @@
 /obj/item/sign/medipolma/examine(mob/user)
 	. = ..()
 	if(claimant)
-		to_chat(user,"This one belongs to Dr.[claimant], MD.")
```
just.. leave it as MD...

HaultyAnonie replied:
it's a cmo specific item.

projectkepler-ru commented on code/game/objects/structures/signs.dm:
```diff
@@ -466,9 +467,14 @@
 
+/obj/item/sign/medipolma/attackby(obj/item/W as obj, mob/user as mob)
+	if(istype(W, /obj/item/tape_roll))
+		user.visible_message(SPAN_WARNING("[user] attaches [W] to the [src]."), SPAN_WARNING("You attach some [W] to the [src]."))
```
what exactly is attaching the sticky tape to the diploma supposed to do?

HaultyAnonie replied:
it's a draft.

Commit:
Update code/game/objects/items/weapons/storage/boxes.dm

personal emergency kit

Co-authored-by: projectkepler-RU <99981766+projectkepler-ru@users.noreply.github.com>

Commit:
Merge branch 'dev' into newsite

Commit:
Update code/modules/projectiles/guns/projectile/pistol.dm

Co-authored-by: projectkepler-RU <99981766+projectkepler-ru@users.noreply.github.com>

Commit:
we got the fiiiirst aidddd yeahh

Commit:
Merge branch 'newsite' of https://github.com/HaultyAnonie/Foundation-19 into newsite

Commit:
balances

cool cool cool.

Commit:
more mapping

expands d-class mining, more ways to escape. less restriction.

Commit:
Merge branch 'dev' into newsite

Commit:
Merge branch 'newsite' of https://github.com/HaultyAnonie/Foundation-19 into newsite

Commit:
mapping + more signs code

Commit:
mapping at 2am got me like

improooovements

Commit:
maps in better flooring, items, decals.

Commit:
fixes a quick single floor tile.

Commit:
spelling mistek ;)

Commit:
fixes the inaccuracy

7u doesnt work for some reason.

Commit:
coroner lockers begone

projectkepler-ru commented:
Stop marking thing as resolved

Tuulipommi commented:
If you keep marking reviews as resolved without actually acting on it, the pr WILL BE CLOSED!

Commit:
Merge branch 'dev' into newsite

HaultyAnonie commented:
> If you keep marking reviews as resolved without actually acting on it, the pr WILL BE CLOSED!
> Stop marking thing as resolved

**IT'S LITERALLY RESOLVED! I RESOLVED IT!!!**
